### PR TITLE
Remove Arbitrum

### PR DESCRIPTION
**Pull request by kovalgek**
#### WHAT/HOW
Remove all contracts/scripts/configs of Arbitrum since this fork is dedicated to Optimism only.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -10,15 +10,11 @@ RPC_ETH_GOERLI=
 RPC_OPT_MAINNET=https://mainnet.optimism.io
 RPC_OPT_GOERLI=https://goerli.optimism.io
```
Comment (TheDZhon) [inline .env.example]:
```suggestion
RPC_OPT_SEPOLIA=https://sepolia.optimism.io
```

Comment (TheDZhon) [reply .env.example]: also, `RPC_ETH_SEPOLIA=` instead of `RPC_ETH_GOERLI=`

File: .env.example
Comment (TheDZhon) [inline .env.example:1]: `NETWORK` also mentiones Görli

File: README.md [hunk 2]
```diff
@@ -259,28 +200,10 @@ Address of the RPC node for **Mainnet** Ethereum network.
...
-
-#### `RPC_ARB_GOERLI`
-
-Address of the RPC node for **Goerli** Arbitrum network.
-
-> Default value: `https://goerli-rollup.arbitrum.io/rpc`
-
 #### `RPC_OPT_GOERLI`
 
 Address of the RPC node for **Goerli** Optimism network.
```
Comment (TheDZhon) [inline README.md]:
```suggestion
Address of the RPC node for **Sepolia** Optimism network.
```

File: README.md [hunk 3]
```diff
@@ -532,14 +404,11 @@ Address of the deployed Governance Bridge Executor in the Optimism network. If s
 
 The private key from the address which holds:
 
-- Goerli and Arbitrum/Optimistic Goerli Ether to launch Arbitrum/Optimism E2E tests
+- Goerli and Optimistic Goerli Ether to launch Optimism E2E tests
```
Comment (TheDZhon) [inline README.md]:
```suggestion
- Goerli and Optimistic Sepolia Ether to launch Optimism E2E tests
```

File: README.md
Comment (TheDZhon) [inline README.md:1]: Let's change Görli to Sepolia

Comment (TheDZhon) [inline README.md:1]: let's change Görli to Sepolia

File: hardhat.config.ts
Comment (TheDZhon) [inline hardhat.config.ts:1]: let's try to remove `0.6.11` from the config

**Review by TheDZhon — CHANGES_REQUESTED** [review]
LGTM except only for a few Görli mentions :tada:

**Review by loga4 — APPROVED** [review]
LGTM

**Review by TheDZhon — APPROVED** [review]
LGTM

**Pull request merged**